components/camel-jaxb/src/test/java/org/apache/camel/jaxb/FallbackTypeConverterTest.java

```diff
@@ -14,21 +14,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.camel.itest.jaxb;
-
-import java.io.InputStream;
```

Comment:
A note: you need to adjust your formatter to match the coding standard used by the project. Imports like this should not come last. The order should be +/-:

``
java.*
javax.* 

all other imports
```

Although much of this is done automatically by the build, it prevents us from having to re-generate the code.